catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/ShareableMetacardSharingPolicyPlugin.java

```diff
@@ -82,12 +83,12 @@
             .collect(Collectors.toSet()));
   }
 
-  private Map<String, Set<String>> getPolicy(WorkspaceMetacardImpl workspace) {
+  private Map<String, Set<String>> getPolicy(ShareableMetacardImpl shareableMetacard) {
```

Comment:
❓ If this gets applied to all metacards (genericized) I wonder if we could roll this in with our other policy impls somehow.

Reply:
Agreed. Should be addressed post-incubation period.